crates/rust-analyzer/src/lib.rs

```diff
@@ -9,6 +9,15 @@
 //! The `cli` submodule implements some batch-processing analysis, primarily as
 //! a debugging aid.
 
+/// Any toolchain less than this version will likely not work with rust-analyzer built from this revision.
+pub const MINIMUM_SUPPORTED_TOOLCHAIN_VERSION: semver::Version = semver::Version {
+    major: 1,
+    minor: 78,
```

Comment:
I kind of arbitrarily picked this fwiw, I think this was the last one we mentioned somewhere before so